website/package.json

```
@@ -81,9 +81,9 @@
     "ember-load-initializers": "^2.1.2",
     "ember-math-helpers": "^3.0.0",
     "ember-meta": "^2.0.0",
-    "ember-modifier": "^4.1.0",
+    "ember-modifier": "^4.2.0",
     "ember-page-title": "^8.2.3",
-    "ember-power-select": "^8.2.0",
+    "ember-power-select": "^8.6.1",
```

They released a new patch recently, but could be bumped in https://github.com/hashicorp/design-system/pull/2612#pullrequestreview-2533912133 too, depending on which one gets in first.

```suggestion
    "ember-power-select": "^8.6.2",
```

---

The other one got in first. I'd appreciate if we can bump this to 8.6.2 (it fixes an accessibility issue)

---

Upgraded to `8.6.2`. Also moved the changeset entry for the upgrade from the super-select to the general list of upgrades.